src/_playground/App.scss

```diff
@@ -84,6 +84,27 @@ $menu-spacing: 20px;
             height: 2rem;
         }
     }
+
+    &__screen-reader-only-focusable {
+        background-color: transparent;
+        color: #fff;
+        position: absolute;
+        width: 1px;
+        height: 1px;
+        margin: -1px;
+        padding: 0;
+        overflow: hidden;
+        border: 0;
+
+        &:focus {
+        position: static;
```

Comment:
I wouldn't have this flow with the other content if it takes focus.  I would set `position: absolute` and then set `top: 0` and `left: 0`.  Although it's already absolutely positioned so maybe remove `position` from this block and just set `top: 0` and `left: 0` at the class level.